manifests/cfcr.yml

```diff
@@ -352,6 +352,7 @@ instance_groups:
         docker-endpoint: unix:///var/vcap/sys/run/docker/docker.sock
         kubeconfig: /var/vcap/jobs/kubelet/config/kubeconfig
         network-plugin: cni
+        root-dir: /var/vcap/data/kubelet
```

Comment:
We should add a unit test (ideally rspec, I'd like to retire the bash test framework someday) that verifies this as the default so that it doesn't get tweaked arbitrarily going forward. There's not a great way to integration-test this behavior (and debatably we'd just be verifying that the Kubelet's flag does what it claims), but it's better than nothing.

Reply:
So, I really hate writing untested code, but in general, if the test I write is basically identical to the code I wrote I don't consider it to be an actual test. It just makes it so that someone who wants to make an arbitrary change has to do it in two places rather than one, there isn't anything that makes them more aware of any negative consequences of that change. We actually started writing a test and then deleted it for this reason. Part of our thinking was that if we test that one line then we should test all the other lines. If we take that reasoning to one logical conclusion we'd end up with a test that literally contains a copy of the file and asserts that the file is the same as that copy. With that in mind do you still want us to write that test? Or a different test?

Reply:
My intent was that the tests would serve as a form of documentation for _why_ that flag is present. Of course one could just modify the thing in two locations, but hopefully the context of the test would at least drive the modification to remain consistent with our purpose.

An integration test to do the same would be quite cumbersome, though if structured well, could be more resilient to arbitrary changes in BOSH's mount scheme. Maybe we should consider adding a test in kubo-ci instead.